functions/helloworld/hello_world.go

```diff
@@ -20,10 +20,16 @@ package helloworld
 import (
 	"fmt"
 	"net/http"
+
+	"github.com/GoogleCloudPlatform/functions-framework-go/functions"
 )
 
-// HelloGet is an HTTP Cloud Function.
-func HelloGet(w http.ResponseWriter, r *http.Request) {
+func init() {
+	functions.HTTP("HelloGet", helloGet)
+}
+
+// helloGet is an HTTP Cloud Function.
+func helloGet(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should the first `h` be capitalized?

Reply:
I believe with this new declarative style these can be private now as they are just registered globally